rnote-ui/src/workspacebrowser/mod.rs

```diff
@@ -302,6 +302,7 @@ impl WorkspaceBrowser {
             filefilter.add_pattern("*.rnote");
             filefilter.add_pattern("*.xopp");
             filefilter.add_pattern("*.svg");
+            filefilter.add_pattern("*.dup");
```

Comment:
Hm.. this kinda bothers me tbh, because you have to know that this is secretly connected to `duplicate.rs` and I don't want to write a comment to "fix" this. We might need to think something else for this.

Reply:
This won't be the case, I implemented the `.dup` to add '.dup` until this file doesn't exist, so it would look like the following:
```
test.xopp
test.xopp.dup // first time
test.xopp.dup.dup // second time
```

Reply:
I can refactor that a bit in another PR.

Reply:
ah, can you please do that in this one? using methods like `.file_stem()`on a path makes it easy. You can look at this very recently [added method](https://github.com/flxzt/rnote/blob/main/rnote-engine/src/utils.rs#L77) for inspiration. Everything else looks fine, and then I am happy.

Reply:
Whoops I forgot to fix this one.... so you'd like to have `.dup` then `.dup2` then `.dup3` and so on?

Reply:
no, like this: `file - dup.pdf`, `file - dup2.pdf` etc. So that the file extension is kept.